app/models/page.rb

```diff
@@ -15,15 +16,15 @@ class Page
   field :role  # only relevant for virtual pages
   field :active, type: Boolean, default: false
   field :layout, type: String, default: 'default'
-  field :meta_description, localize: Slices::Config.i18n?
-  field :title, localize: Slices::Config.i18n?
+  field :meta_description, localize: Slices::Config.i18n?, default: ''
+  field :title, localize: Slices::Config.i18n?, default: ''
```

Comment:
I started to look into this, but I didn't find anything conclusive. The follow spec fails if the above defaults are missing:

```ruby
rspec spec/apis/pages/update_spec.rb:133
```